tinker_cookbook/tests/test_renderers.py

```diff
@@ -44,12 +45,80 @@ def test_against_hf_chat_templates(model_name: str):
     elif model_name.startswith("deepseek-ai"):
         aug_convo = convo
     elif model_name.startswith("openai"):
+        convo[1] = {
```

Comment:
Maybe just set `convo[1]["thinking"] = "..."`
Also, I think it's not actually tested here since it's not the last message so we don't render the thinking?

Reply:
Testing it's not rendered is good, but we should also test case where it is rendered.
Let's maybe add a 4th message to the convo where the assistant answers the question with thinking text, and then run the test with and without the last message i.e. on `convo[:-1]` and `convo`

Reply:
Right sorry, I missed the separate supervised test, looks good then!